spec/lib/tasks/correct_address_from_csv_spec.rb

```diff
@@ -2,40 +2,60 @@
 require "rake"
 
 RSpec.describe "data_import" do
-  def replace_entity_ids(lettings_log, second_lettings_log, third_lettings_log, export_template)
-    export_template.sub!(/\{id\}/, lettings_log.id.to_s)
-    export_template.sub!(/\{id2\}/, second_lettings_log.id.to_s)
-    export_template.sub!(/\{id3\}/, third_lettings_log.id.to_s)
+  def replace_entity_ids(log, second_log, third_log, fourth_log, export_template)
+    export_template.sub!(/\{id\}/, log.id.to_s)
+    export_template.sub!(/\{id2\}/, second_log.id.to_s)
+    export_template.sub!(/\{id3\}/, third_log.id.to_s)
+    export_template.sub!(/\{id4\}/, fourth_log.id.to_s)
   end
 
-  describe ":import_address_from_csv", type: :task do
-    subject(:task) { Rake::Task["data_import:import_address_from_csv"] }
+  before do
+    allow(Storage::S3Service).to receive(:new).and_return(storage_service)
+    allow(Configuration::EnvConfigurationService).to receive(:new).and_return(env_config_service)
+    allow(Configuration::PaasConfigurationService).to receive(:new).and_return(paas_config_service)
+    allow(ENV).to receive(:[])
+    allow(ENV).to receive(:[]).with("IMPORT_PAAS_INSTANCE").and_return(instance_name)
+    allow(ENV).to receive(:[]).with("VCAP_SERVICES").and_return("dummy")
+
+    WebMock.stub_request(:get, /api.postcodes.io/)
```

Comment:
can we just do `\.` to avoid this check failure in the first place rather than overriding it?

Reply:
I accept it's only a test but it could get copied into live code by a confused developer later down the line (can't really imagine how but in principle I'd rather fix vulnerabilities than rely on manually overriding them but that's just my preference, happy if you are

Reply:
Good point